tests/11-getting-started.sh

```diff
@@ -27,7 +26,7 @@ echo "------ checking cilium status ------"
 cilium status
 
 echo "------ creating Docker network of type Cilium ------"
```

Comment:
Remove the echo statements prior to the calls to this helper function as the echo statement is already part of the helper function.

Reply:
Was actually done earlier but probably pushed from the wrong machine. Should be gone now.